albumentations/augmentations/transforms.py

```diff
@@ -3320,3 +3322,91 @@ def get_transform_init_args_names(self):
             "mask_fill_value",
             "random_offset",
         )
+
+
+class Defocus(ImageOnlyTransform):
+    """
+    Apply defocus transform. See arXiv:1903.12261 [cs.LG].
+
+    Args:
+        radius ((int, int) or int): range for radius of defocusing.
+            If limit is a single int, the range will be [1, limit]. Default: (3, 10).
+        alias_blur ((float, float) or float): range for alias_blur of defocusing (sigma of gaussian blur).
+            If limit is a single float, the range will be (0, limit). Default: (0.1, 0.5).
+        p (float): probability of applying the transform. Default: 0.5.
+
+    Targets:
+        image
+
+    Image types:
+        Any
+    """
+
+    def __init__(self, radius=(3, 10), alias_blur=(0.1, 0.5), always_apply=False, p=0.5):
+        super(Defocus, self).__init__(always_apply, p)
+        self.radius = to_tuple(radius, low=1)
+        self.alias_blur = to_tuple(alias_blur, low=0)
+
+        if self.radius[0] < 0:
+            raise ValueError("Parameter radius must be positive")
+
+        if self.alias_blur[0] <= 0:
+            raise ValueError("Parameter alias_blur must be non-negative")
+
+    def apply(self, img, radius=3, alias_blur=0.5, **params):
+        return F.defocus(img, radius, alias_blur)
+
+    def get_params(self):
+        return {
+            "radius": random.choice(np.arange(self.radius[0], self.radius[1] + 1)),
+            "alias_blur": random.uniform(self.alias_blur[0], self.alias_blur[1]),
+        }
+
+    def get_transform_init_args_names(self):
+        return ("radius", "alias_blur")
+
+
+class ZoomBlur(ImageOnlyTransform):
+    """
+    Apply zoom blur transform. See arXiv:1903.12261 [cs.LG].
+
+    Args:
+        max_factor ((float, float) or float): range for max factor for blurring.
+            If max_factor is a single float, the range will be (1, limit). Default: (1, 1.31).
+            All max_factor values should be larger than 1.
+        step_factor ((float, float) or float): If single float will be used as step parameter for np.arange.
+            If tuple of float step_factor will be in range `[step_factor[0], step_factor[1])`. Default: (0.01, 0.03).
+            All step_factor values should be positive.
+        p (float): probability of applying the transform. Default: 0.5.
+
+    Targets:
+        image
+
+    Image types:
+        Any
+    """
+
+    def __init__(self, max_factor=1.31, step_factor=(0.01, 0.03), always_apply=False, p=0.5):
+        super().__init__(always_apply, p)
+        self.max_factors = to_tuple(max_factor, low=1.0)
+        self.step_factors = to_tuple(step_factor, step_factor)
+
+        if self.max_factors[0] < 1:
+            raise ValueError("Max factor must be larger or equal 1")
```

Comment:
I think this check is redundant because you described the minimum values ​​in to_tuple